integration-tests/mysql_test.go

```diff
@@ -5,29 +5,47 @@ import (
 	"path/filepath"
 	"regexp"
 	"runtime"
+	"strings"
 
 	testframework "github.com/cloudfoundry/cloud-service-broker/brokerpaktestframework"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
+var customMySQLPlan = map[string]interface{}{
+	"name":                  "custom-plan",
+	"id":                    "9daa07f1-78e8-4bda-9efe-91576102c30d",
+	"description":           "custom plan defined by customer",
+	"display_name":          "custom plan defined by customer (beta)",
+	"mysql_version":         "MYSQL_5_7",
+	"credentials":           "plan_cred",
+	"project":               "plan_project",
+	"authorized_network":    "plan_authorized_network",
+	"authorized_network_id": "plan_authorized_network_id",
+	"require_ssl":           false,
+}
+
 var _ = Describe("Mysql", func() {
 	AfterEach(func() {
 		Expect(mockTerraform.Reset()).NotTo(HaveOccurred())
 	})
 
 	It("should publish mysql in the catalog", func() {
-		catalog, err := broker.Catalog()
+		expectedPlans := []string{"small", "medium", "large", customMySQLPlan["name"].(string)}
 
-		Expect(err).NotTo(HaveOccurred())
+		catalog, err := broker.Catalog()
+		Expect(err).ShouldNot(HaveOccurred())
 
 		service := testframework.FindService(catalog, "csb-google-mysql")
-		Expect(service.Plans).To(HaveLen(3))
+		Expect(service.Plans).To(HaveLen(4))
 		Expect(service.ID).ShouldNot(BeNil())
 		Expect(service.Name).ShouldNot(BeNil())
 		Expect(service.Tags).Should(ConsistOf([]string{"gcp", "mysql", "beta"}))
 		Expect(service.Metadata.ImageUrl).ShouldNot(BeNil())
 		Expect(service.Metadata.DisplayName).ShouldNot(BeNil())
+		for _, plan := range service.Plans {
```

Comment:
Thank you! I'll analyse the alternatives you mentioned

Reply:
Good afternoon @blgm , I have not found a good way to compare the arrays due to the type of the array ( `domain.ServicePlan` ). It is a struct with a high number of properties and I have not known how to compare only one of them. It’s probably because of my lack of knowledge of Omega and Ginko. Finally I have implemented another solution that has led me to discover that the metadata of the custom plan was badly defined, which makes me think that the same thing happens with the Postgres plan.

I'd like to know your thoughts on what I've implemented and if it adds value. From my perspective as a novice, it has it because it serves as documentation, but I prefer to know if you would delete it or there is another better alternative.

Thank you very much for the effort!